src/writeFile.js

```diff
@@ -71,6 +71,7 @@ export default function writeFile(globalRef, pattern, file) {
                 // A hack so .dotted files don't get parsed as extensions
                 let basename = path.basename(file.relativeFrom);
                 let dotRemoved = false;
+                let slashAdded = false;
```

Comment:
Can you elaborate why this is needed ?

Reply:
Because the slash is removed under certain circumstances. It's documented within the source of that file. (I'm on my phone right now. I can explain it in more detail later if necessary, but I think you can find the reason already documented in the file.)

Reply:
Sorry, I said "the slash is removed," but I meant a leading slash is *added*. Anyway,  yeah, the reason for adding the leading slash is [explained within the file](https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/writeFile.js#L85-L90). The reason I remove it again is to ensure that the manifest keys are consistent.

Honestly, I'd say that `file.relativeFrom` should go completely unaltered in that `(pattern.toType === 'template')` block, and a new variable should instead be created and passed to `loaderUtils.interpolateName()`. I didn't do that here since that would be a refactor, which I think would be outside the scope of this PR.

Reply:
All good, that's obviously on our side then :). Could you please add a comment with the link `https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/writeFile.js#L85-L90` as an explanation/context for why this is needed atm?